### ISSUE TITLE
[SCREENREADER]: For publicly available content, focus SHOULD start at the top of the page, not the breadcrumbs

### ISSUE THREAD

**Issue by jenstrickland** [department-of-veterans-affairs/va.gov-team#6635]
**Feedback framework**

```diff
-  **Must** for if the feedback must be applied 
!  **Should** if the feedback is best practice 
+ ##Consider** for suggestions/enhancements 
```
## Description

As a screenreader user, I **SHOULD** have a consistent experience when arriving on pages, the content should start at the top, and offer the Skip to content link. WCAG 2.0 Level A, Success Criteria 2.4.3 states: 
> If a Web page can be navigated sequentially and the navigation sequences affect meaning or operation, focusable components receive focus in an order that preserves meaning and operability. (Level A)

Using JAWS + IE11, for the Disability benefits hub page started at the top of the page, offered the Skip to content link, however, the How to file a disability claim starts on the breadcrumbs. I tested with VoiceOver + Safari, both start at the top of the page, offer the Skip to content link. According to a web survey, JAWS+IE11 is the most used screenreader and browser pairing.

- Disability benefits hub: https://www.va.gov/disability/
- How to file a disability claim: https://www.va.gov/disability/how-to-file-claim/

For publicly available pages that may be accessed via web search results, screenreader users will expect the pages to consistently start reading at the top of the page. When the page begins on the breadcrumbs, there is no context for the content being read, and it is disorienting.

**Note:** To fix this issue, the developer will need to pair with the accessibility specialist to validate in JAWS+IE11.

## Point of Contact

**VFS Point of Contact:** Jennifer

## Acceptance Criteria

As a screenreader user, I want to have a consistent experience when arriving on pages, the content should start at the top, and offer the Skip to content link.

## Environment

* Operating System: Windows 10 Pro, version 1809
* Browser: IE 11.1039
* Screenreading device: JAWS
* Server destination: production

## Steps to Recreate

1. Enter the urls listed in browser
    - Disability benefits hub: https://www.va.gov/disability/
    - How to file a disability claim: https://www.va.gov/disability/how-to-file-claim/
2. Start screenreading device listed in Environment
3. Reload the page
4. Verify that the hub starts reading at the top of the page, and for the How to file a disability claim the focus is on the breadcrumbs

## WCAG or Vendor Guidance (optional)

* [WCAG 2.4.3 Focus Order](https://www.w3.org/TR/UNDERSTANDING-WCAG20/navigation-mechanisms-focus-order.html)

**Comment by jenstrickland** [comment]
I wanted to validate this behavior with someone else with JAWS+IE11, and am unable to replicate on another machine, so I'm closing this ticket.
